packages/contracts/contracts/optimistic-ethereum/ovm/decompression/SequencerMessageDecompressor.sol

```diff
@@ -0,0 +1,134 @@
+pragma solidity ^0.5.0;
+
+/* Library Imports */
+import { BytesLib } from "../../utils/libraries/BytesLib.sol";
+import { DataTypes } from "../../utils/libraries/DataTypes.sol";
+import { TransactionParser } from "../../utils/libraries/TransactionParser.sol";
+import { ECDSAUtils } from "../../utils/libraries/ECDSAUtils.sol";
+import { ExecutionManagerWrapper } from "../../utils/libraries/ExecutionManagerWrapper.sol";
+
+/* Contract Imports */
+import { ExecutionManager } from "../ExecutionManager.sol";
+
+/**
+ * @title SequencerMessageDecompressor
+ */
+contract SequencerMessageDecompressor {
+    /*
+     * Data Structures
+     */
+    
+    enum TransactionType {
+        EOA_CONTRACT_CREATION,
+        NATIVE_ETH_TRANSACTION,
```

Comment:
What do you think about calling this `EIP155` instead of `NATIVE_ETH_TRANSACTION`? I think its more descriptive of its type